nrf_802154/doc/feature_description.rst

```diff
@@ -361,3 +357,10 @@ The higher layer can implement various retransmission schemes by combining the i
    If the higher layer marks a frame as already secured, it must not expect the driver to update the dynamic data of the frame.
    Transmitting an encrypted frame with its header modified afterward results in a security breach.
    An attempt to transmit a frame with such parameters will fail unconditionally.
+
+Supporting Front-End Modules
+****************************
+
+The driver supports Front-End Modules (FEMs) using an implementation provided by the `mpsl`_.
```

Comment:
Documentation build [fails on this line](https://github.com/nrfconnect/sdk-nrf/runs/6589279046?check_suite_focus=true) with the following error:
```
/home/runner/work/sdk-nrf/sdk-nrf/ncs/nrf/doc/_build/nrfxlib/src/nrf_802154/doc/feature_description.rst:364:Unknown target name: "mpsl".
```

Reply:
pushed fix already

Reply:
@anangl doc built correctly